src/swsssdk/scripts/sonic-db-cli

```diff
@@ -34,8 +52,11 @@ def ping_all_instances():
         print 'PONG'
         sys.exit(0)
 
-def execute_cmd(dbname, cmd):
-    dbconn = swsssdk.SonicV2Connector(use_unix_socket_path=False)
+def execute_cmd(dbname, cmd, namespace):
+    if namespace is None:
```

Comment:
>if namespace is None: [](start = 4, length = 21)

I see the intelligence to auto choosing a use_unix_socket_path based on use case. Thanks!

It will be useful to provide another command line argument `-s/--unix-socket` to let use override this choosing.

Reply:
Updated with a new option.